packages/components/src/Alert/__tests__/__snapshots__/Alert.test.js.snap

```diff
@@ -605,6 +605,7 @@ exports[`props should render as dismissable 1`] = `
           data-g2-component="CloseButton"
           data-icon="true"
           title="Dismiss"
+          type="button"
```

Comment:
That's a lot of unnecessary snapshot files! Maybe we should just snapshot this at most once for each component and do the rests with normal asserting?

Ref: https://kentcdodds.com/blog/effective-snapshot-testing

Reply:
@kevin940726 Great observation! For context, a majority of these tests were added as a "better than nothing" solution during the initial rapid development phase for this project :).

@sarayourfriend and I have been refactoring these as [we've migrated it over to Gutenberg](https://github.com/WordPress/gutenberg/issues/28399).

Speaking of which, we'll need to make this update in Gutenberg 👍